Reflections/Week_4_reflection.md

```diff
@@ -0,0 +1,15 @@
+## Accomplishments
+
+Ultimately this week was a week of decision paralysis. The roadblock that I had faced with implementing the iterator syntax was enough to completely freeze me from getting any meaningful work done. I am also writing this reflection late, as a result of this mental block.
```

Comment:
I hadn't been able to put a phrase to it, but "decision paralysis" sums this up perfectly-- it's not an easy thing to work through, and I'm very glad you've been able to overcome it!